app/controllers/ops_controller/settings/help_menu.rb

```diff
@@ -4,21 +4,19 @@ module OpsController::Settings::HelpMenu
   def settings_update_help_menu
     return unless load_edit('customize_help_menu')
 
-    konfig = VMDB::Config.new("vmdb")
-
     begin
-      konfig.config = Vmdb::Settings.decrypt_passwords!(Settings.to_hash)
-      konfig.config[:help_menu].merge!(@edit[:new])
-      konfig.validate
+      konfig = Vmdb::Settings.decrypt_passwords!(Settings.to_hash)
+      konfig[:help_menu].merge!(@edit[:new])
+      success, config_errors = Vmdb::Settings.validate(konfig)
```

Comment:
It seems to me that this `sucess` here is not used, if it's true please change it to `_`.